### ISSUE TITLE
Cache getter implementation

### ISSUE THREAD

**Issue by feynmansfedora** [feynmansfedora/ywot-bot#12]
In branch `cache`, getters have been implemented which request an area from the internal data of `cachespace`/check if it exists. Now, those getters have to be implemented into other parts of `World`. Specifically, the fetch function should have a conditional that returns getcached if true or makes the request like normal otherwise.
```javascript
cache = getcached(minY,minX,maxY,maxX);
if (cache) return cache;
```
works to skip the rest of the function.

**Comment by feynmansfedora** [comment]
It's been implemented successfully in branch `cache`.

**Comment by feynmansfedora** [comment]
This can only be implemented after #13 (because it is pointless otherwise).
